app/assets/stylesheets/components/_card_list.css.sass

```diff
@@ -26,6 +26,13 @@
       &.ng-enter-active
         animation-name: bounceInLeft
 
+    .CardList-card.is-closed:nth-child(#{$i})
```

Comment:
Fixing it. Also, the hover opacity effect should only apply to elements with `.CardList-card.is-closed` classes.